mqtt-examples/src/main/java/io/vertx/example/mqtt/app/Client.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2016 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.vertx.example.mqtt.app;
+
+import io.netty.handler.codec.mqtt.MqttQoS;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.example.mqtt.util.Runner;
+import io.vertx.mqtt.MqttClient;
+import io.vertx.mqtt.MqttClientOptions;
+
+import java.nio.charset.Charset;
+
+/**
+ * An example of using the MQTT client
+ */
+public class Client extends AbstractVerticle {
+
+  private static final String MQTT_TOPIC = "/my_topic";
+  private static final String MQTT_MESSAGE = "Hello Vert.x MQTT Client";
+  private static final String SERVER_HOST = "0.0.0.0";
```

Comment:
It's about the client connecting to a server so it should be an IP address (of course "localhost" or 127.0.0.1 is ok) but not 0.0.0.0 which means all IP addresses on the local machine (it's used for binding the server listening). At same time, it's better to call it BROKER_HOST. When people look for MQTT client examples they want to connect to MQTT broker in general.